mux.PGO.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <PlatformShortName Condition="'$(PlatformShortName)'==''">$(Platform)</PlatformShortName>
+        <PGOFilesDir>$(VCToolsInstallDir)bin\Host$(PlatformShortName)\$(PlatformShortName)</PGOFilesDir>
+        <PGOFilesDir Condition="'$(PlatformShortName)'=='ARM' or '$(PlatformShortName)'=='ARM64'">$(VCToolsInstallDir)bin\$(PlatformShortName)</PGOFilesDir>       
+    </PropertyGroup>
+    <PropertyGroup>    
+        <PGOBuildMode Condition="'$(PGOBuildMode)'==''">Optimize</PGOBuildMode>
```

Comment:
>Optimize [](start = 56, length = 8)

we could probably do with not having this option and always optimize if the profile guided database is available.

Reply:
This seems useful when building locally to quickly enable/disable optimizations. So decided to keep it.

---
In reply to: [300175577](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r300175577) [](ancestors = 300175577)